siliconcompiler/utils/logging.py

```
@@ -5,6 +5,51 @@
+    def __init__(self, logger: logging.Logger, skip: logging.Handler = None):
+        super().__init__()
+        self._logger = logger
+        self._skip = skip
+
+    def emit(self, record):
+        for handler in list(self._logger.handlers):
+            if handler is self._skip:
+                continue
+            try:
+                handler.handle(record)
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)